### PR TITLE
25.0.0 beta 1

### PR DESCRIPTION
**Pull request by skjnldsv**
# Pending PRs:
* [ ] #17425 @xklonx
* [ ] #22598 @dassio
* [ ] #23036 @chrisfritsche
* [ ] #23695 @rullzer
* [ ] #23825 @rullzer
* [ ] #23914 @rullzer
* [ ] #24574 @zenlord
* [ ] #24579 @danxuliu
* [ ] #24596 @kofemann
* [ ] #25023 @RafaOP
* [ ] #25109 @icewind1991
* [ ] #25192 @blizzz
* [ ] #25471 @rullzer
* [ ] #25551 @rullzer
* [ ] #25768 @icewind1991
* [ ] #26155 @blizzz
* [ ] #26178 @ArtificialOwl
* [ ] #26318 @icewind1991
* [ ] #26320 @icewind1991
* [ ] #26358 @skjnldsv
* [ ] #26408 @kevin147147
* [ ] #26411 @skjnldsv
* [ ] #26496 @christlang
* [ ] #26716 @skjnldsv
* [ ] #26871 @brueckner
* [ ] #26940 @rullzer
* [ ] #27034 @juliushaertl
* [ ] #27119 @ArtificialOwl
* [ ] #27398 @skjnldsv
* [ ] #27492 @cyclops8456
* [ ] #27538 @icewind1991
* [ ] #27562 @j-be
* [ ] #27613 @Dreamsorcerer
* [ ] #27633 @ArtificialOwl
* [ ] #27806 @p5n
* [ ] #27820 @juliushaertl
* [ ] #27961 @CarlSchwan
* [ ] #28019 @Pytal
* [ ] #28226 @gary-kim
* [ ] #28236 @ArtificialOwl
* [ ] #28279 @J0WI
* [ ] #28310 @juliushaertl
* [ ] #28311 @juliushaertl
* [ ] #28601 @juliushaertl
* [ ] #28629 @MasahikoSada
* [ ] #28635 @TSI-yogeshshejwadkar
* [ ] #28636 @TSI-yogeshshejwadkar
* [ ] #28645 @TSI-kavitasonawane
* [ ] #28695 @MasahikoSada
* [ ] #28772 @icewind1991
* [ ] #28832 @TSI-yogeshshejwadkar
* [ ] #28834 @TSI-kavitasonawane
* [ ] #28911 @danxuliu
* [ ] #29010 @mattian
* [ ] #29143 @eneiluj
* [ ] #29368 @philipgatzka
* [ ] #29430 @CarlSchwan
* [ ] #29642 @szaimen
* [ ] #29643 @szaimen
* [ ] #29729 @eneiluj
* [ ] #29861 @Zocker1999NET
* [ ] #29881 @CarlSchwan
* [ ] #29892 @Pytal
* [ ] #29929 @SpamReceiver
* [ ] #29940 @jonathanmmm
* [ ] #29988 @MichaIng
* [ ] #30147 @Phlogi
* [ ] #30183 @Pytal
* [ ] #30223 @CarlSchwan
* [ ] #30359 @juliushaertl
* [ ] #30366 @marcelklehr
* [ ] #30412 @quentinguidee
* [ ] #30472 @juliushaertl
* [ ] #30498 @juliushaertl
* [ ] #30512 @come-nc
* [ ] #30586 @solracsf
* [ ] #30608 @solracsf
* [ ] #30613 @blizzz
* [ ] #30654 @pavelkryl
* [ ] #30777 @AsamK
* [ ] #30796 @icewind1991
* [ ] #30884 @ArtificialOwl
* [ ] #30895 @mdoggydog
* [ ] #30957 @rotdrop
* [ ] #30972 @icewind1991
* [ ] #30980 @kominoshja
* [ ] #30986 @rotdrop
* [ ] #31001 @takahiro-blab
* [ ] #31038 @itmanagerro
* [ ] #31064 @tcitworld
* [ ] #31074 @artonge
* [ ] #31159 @chuckdries
* [ ] #31160 @CarlSchwan
* [ ] #31286 @rotdrop
* [ ] #31346 @CarlSchwan
* [ ] #31386 @artonge
* [ ] #31396 @skjnldsv
* [ ] #31469 @nextcloud-command
* [ ] #31499 @juliushaertl
* [ ] #31571 @michielbdejong
* [ ] #31660 @dzatoah
* [ ] #31699 @CommanderRoot
* [ ] #31790 @tcitworld
* [ ] #31793 @CarlSchwan
* [ ] #31807 @jotoeri
* [ ] #31859 @max65482
* [ ] #31905 @icewind1991
* [ ] #31938 @szaimen
* [ ] #31944 @PVince81
* [ ] #31947 @andyxheli
* [ ] #32006 @thomasrockhu-codecov
* [ ] #32012 @CarlSchwan
* [ ] #32018 @CarlSchwan
* [ ] #32019 @CarlSchwan
* [ ] #32028 @icewind1991
* [ ] #32032 @Valdnet
* [ ] #32038 @nickvergessen
* [ ] #32163 @icewind1991
* [ ] #32197 @blizzz
* [ ] #32211 @artonge
* [ ] #32226 
* [ ] #32231 @skjnldsv
* [ ] #32257 @artonge
* [ ] #32314 @icewind1991
* [ ] #32321 @tcitworld
* [ ] #32343 @skjnldsv
* [ ] #32400 @skjnldsv
* [ ] #32440 @carlcsaposs
* [ ] #32441 
* [ ] #32461 @icewind1991
* [ ] #32466 @Pytal
* [ ] #32499 @eneiluj
* [ ] #32523 @carlcsaposs
* [ ] #32524 @carlcsaposs
* [ ] #32530 
* [ ] #32562 @icewind1991
* [ ] #32565 @ChristophWurst
* [ ] #32631 @ASerbinski
* [ ] #32632 @icewind1991
* [ ] #32635 @andyxheli
* [ ] #32739 @artonge
* [ ] #32740 @jasperweyne
* [ ] #32761 @nickvergessen
* [ ] #32775 @Pytal
* [ ] #32780 @ChristophWurst
* [ ] #32781 @unteem
* [ ] #32787 @jcjveraa
* [ ] #32814 @danxuliu
* [ ] #32877 @icewind1991
* [ ] #32899 @Retidurc
* [ ] #32912 @CarlSchwan
* [ ] #32943 @icewind1991
* [ ] #32961 @CarlSchwan
* [ ] #33047 @CarlSchwan
* [ ] #33061 @pboguslawski
* [ ] #33087 @szaimen
* [ ] #33091 
* [ ] #33104 @miaulalala
* [ ] #33105 @nickvergessen
* [ ] #33106 @vitormattos
* [ ] #33161 @vincentdh
* [ ] #33177 @Pytal
* [ ] #33203 @solracsf
* [ ] #33217 @Pytal
* [ ] #33221 @tcitworld
* [ ] #33239 @juliushaertl
* [ ] #33259 @CarlSchwan
* [ ] #33285 @Pytal
* [ ] #33292 @icewind1991
* [ ] #33305 @LinneyS
* [ ] #33310 @Pytal
* [ ] #33311 @Pytal
* [ ] #33322 @PVince81
* [ ] #33332 @danxuliu
* [ ] #33347 @max-nextcloud
* [ ] #33375 @tcitworld
* [ ] #33381 @luzpaz
* [ ] #33393 @luzpaz
* [ ] #33396 @Xavierando
* [ ] #33397 @cneukom
* [ ] #33409 @Pytal
* [ ] #33429 @CarlSchwan
* [ ] #33430 @come-nc
* [ ] #33433 @come-nc
* [ ] #33434 @Jerome-Herbinet
* [ ] #33457 @CarlSchwan
* [ ] #33458 @icewind1991
* [ ] #33470 @SpamReceiver
* [ ] #33482 @juliushaertl
* [ ] #33484 @danxuliu
* [ ] #33495 @danxuliu
* [ ] #33500 @icewind1991
* [ ] #33502 @Pytal
* [ ] #33511 @artonge
* [ ] #33512 @Pytal
* [ ] #33516 @PVince81
* [ ] #33518 @icewind1991
* [ ] nextcloud/3rdparty#1000 
* [ ] nextcloud/3rdparty#1009 @szaimen
* [ ] nextcloud/3rdparty#1019 
* [ ] nextcloud/3rdparty#1020 
* [ ] nextcloud/3rdparty#697 
* [ ] nextcloud/3rdparty#710 
* [ ] nextcloud/3rdparty#865 
* [ ] nextcloud/3rdparty#883 
* [ ] nextcloud/3rdparty#899 
* [ ] nextcloud/3rdparty#902 @kvikshaug
* [ ] nextcloud/3rdparty#922 
* [ ] nextcloud/3rdparty#997 @vitormattos
* [ ] nextcloud/activity#327 @tinko92
* [ ] nextcloud/activity#460 @madmas
* [ ] nextcloud/activity#504 @chrisfritsche
* [ ] nextcloud/activity#530 @tintou
* [ ] nextcloud/activity#589 @artonge
* [ ] nextcloud/activity#646 @TSI-nareshrai
* [ ] nextcloud/activity#693 @fschrempf
* [ ] nextcloud/activity#698 @icewind1991
* [ ] nextcloud/activity#861 @CarlSchwan
* [ ] nextcloud/circles#190 @Shinrai
* [ ] nextcloud/circles#226 @fgsl
* [ ] nextcloud/circles#229 @fgsl
* [ ] nextcloud/circles#338 @hoxia
* [ ] nextcloud/circles#384 @tortuetorche
* [ ] nextcloud/circles#392 @tortuetorche
* [ ] nextcloud/circles#395 @tortuetorche
* [ ] nextcloud/circles#398 @tortuetorche
* [ ] nextcloud/circles#399 @tortuetorche
* [ ] nextcloud/circles#400 @tortuetorche
* [ ] nextcloud/circles#401 @tortuetorche
* [ ] nextcloud/circles#878 @ArtificialOwl
* [ ] nextcloud/circles#891 @ArtificialOwl
* [ ] nextcloud/circles#966 @ArtificialOwl
* [ ] nextcloud/circles#971 @rexbron
* [ ] nextcloud/circles#980 @ArtificialOwl
* [ ] nextcloud/circles#992 @solracsf
* [ ] nextcloud/example-files#19 @jospoortvliet
* [ ] nextcloud/files_pdfviewer#202 @skjnldsv
* [ ] nextcloud/files_pdfviewer#620 @skjnldsv
* [ ] nextcloud/files_rightclick#129 @FreesphereTom
* [ ] nextcloud/files_rightclick#135 @Cdisk2022
* [ ] nextcloud/files_rightclick#62 @dnviti
* [ ] nextcloud/files_videoplayer#115 @KiritoStudio
* [ ] nextcloud/files_videoplayer#196 @0xpr03
* [ ] nextcloud/files_videoplayer#249 @TSI-sangramdesai
* [ ] nextcloud/files_videoplayer#251 
* [ ] nextcloud/firstrunwizard#610 @mcnesium
* [ ] nextcloud/logreader#476 @Abijeet
* [ ] nextcloud/notifications#728 @nickvergessen
* [ ] nextcloud/notifications#939 @nickvergessen
* [ ] nextcloud/photos#1027 @TSI-ashishsharma
* [ ] nextcloud/photos#1062 @marcelklehr
* [ ] nextcloud/photos#1142 @artonge
* [ ] nextcloud/photos#466 @dassio
* [ ] nextcloud/photos#502 @Mikescops
* [ ] nextcloud/photos#515 @Mikescops
* [ ] nextcloud/photos#607 @cnmicha
* [ ] nextcloud/photos#649 @DeJQit
* [ ] nextcloud/recommendations#354 @blkqi
* [ ] nextcloud/recommendations#458 @sualko
* [ ] nextcloud/serverinfo#194 @hackerESQ
* [ ] nextcloud/serverinfo#257 @sausix
* [ ] nextcloud/serverinfo#264 @dertuxmalwieder
* [ ] nextcloud/serverinfo#267 @akhil1508
* [ ] nextcloud/serverinfo#292 
* [ ] nextcloud/serverinfo#301 
* [ ] nextcloud/serverinfo#331 @overhacked
* [ ] nextcloud/text#2163 @rotdrop
* [ ] nextcloud/text#2378 @eneiluj
* [ ] nextcloud/text#2794 @juliushaertl
* [ ] nextcloud/text#2795 @juliushaertl
* [ ] nextcloud/updater#312 @fermulator
* [ ] nextcloud/updater#317 
* [ ] nextcloud/updater#361 
* [ ] nextcloud/updater#420 @solracsf
* [ ] nextcloud/updater#90 @dlichtenberger
* [ ] nextcloud/viewer#1063 @ktprograms
* [ ] nextcloud/viewer#1242 @vanpertsch
* [ ] nextcloud/viewer#1321 @szaimen
* [ ] nextcloud/viewer#15 @skjnldsv
* [ ] nextcloud/viewer#650 @skjnldsv
* [ ] nextcloud/viewer#915 @skjnldsv

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -30,10 +30,10 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = [25, 0, 0, 4];
+$OC_Version = [25, 0, 0, 5];
 
 // The human readable string
-$OC_VersionString = '25.0.0 dev';
+$OC_VersionString = '25.0.0 beta 1';
```
Comment (nickvergessen) [inline version.php:36]: I was about to comment that we usually space: https://github.com/nextcloud/server/commit/b2b72cb5da4c05d67287eacaea439344ec407181

But now it's fixed already :P

**Review by nickvergessen — APPROVED** [review]

File: version.php (hunk 1 above)
Comment (skjnldsv) [reply version.php:36]: Yes, it's really frustrating! Sometimes we do, sometimes we don't, sometimes the command needs a leading `v`, sometimes it doesn't... 😩

**Review by AndyScherzinger — APPROVED** [review]

**Comment by skjnldsv** [comment]
Scheduled for tomorrow, we need the last @nextcloud/vue

**Comment by PVince81** [comment]
- [ ] nextcloud-vue update to 6.0.0-beta.1 @pvince81

**Pull request merged**